### DISCUSSION TITLE
Is there a BackstopJS style HTML viewer after tests have run?

### DISCUSSION THREAD

**Discussion by d3vl1m3** [oblador/loki#389] [Q&A] [answered]
Hiya all,

BackstopJs have a nice feature where after the PL has ran it creates a html atrifact that you can see the ref, test and diff images in a simple page.

Does Loki have anything similar?

Cheers
Liam

**Comment by techeverri** [comment, marked as answer]
Check @priomsrb's suggestion https://github.com/oblador/loki/issues/76 using [reg-cli](https://github.com/reg-viz/reg-cli)

**Reply by d3vl1m3** [reply]
Great! Thanks for this. Is it worthwhile adding some kind of 'how to' to the docs for this? Imagine a lot of people would like to be able to do this.

**Reply by d3vl1m3** [reply]
Wow - that was stupidly easy to setup. Literally installed it and ran the command (pointing the relevant commands at the dir paths).

Ran it and reg-cli worked first time... I... never even knew that was possible... it's... so beautiful 😭😭
